src/main/java/io/hops/hopsworks/expat/migrations/featurestore/statistics/StatisticsMigration.java

```diff
@@ -331,11 +362,8 @@ private void migrateFeatureDescriptiveStatistics(ResultSet fdsResultSet)
           LOGGER.info(String.format("[migrateFeatureDescriptiveStatistics] Delete FGS ACTIVITY: %s",
             deleteFgsActStmt.toString()));
         } else {
-          LOGGER.info(String.format("[migrateFeatureDescriptiveStatistics] Delete FGS: %s", deleteFgsStmt.toString()));
-          LOGGER.info(String.format("[migrateFeatureDescriptiveStatistics] Delete FGS ACTIVITY: %s",
-            deleteFgsActStmt.toString()));
-          deleteFgsStmt.executeBatch();
-          deleteFgsActStmt.executeBatch();
+          deleteStatisticsBatch(deleteFgsStmt, deleteFGStatisticsIds, "FGS");
+          deleteStatisticsBatch(deleteFgsActStmt, deleteFGStatisticsIds, "FGS ACTIVITY");
```

Comment:
Now that I see it, activities should get deleted automatically when we delete the corresponding statistics. We set `ON DELETE CASCADE` on the statistics id column. It's not harmful, but I don't think the delete operation will delete anything.